### PR TITLE
Document standard backends in matplotlib.use()

### PR DESCRIPTION
**Pull request by timhoffm**

### CODE REVIEW

**Comment by timhoffm** [comment]
Link to backends added.

File: lib/matplotlib/__init__.py [hunk 1]
```diff
@@ -1311,16 +1311,23 @@ def use(arg, warn=True, force=False):
...
+
+        - interactive backends:
+          GTK3Agg, GTK3Cairo, MacOSX, nbAgg,
+          Qt4Agg, Qt4Cairo, Qt5Agg, Qt5Cairo,
+          TkAgg, TkCairo, WebAgg, WX, WXAgg, WXCairo
+
+        - non-interactive backends:
+          agg, cairo, pdf, pgf, ps, svg, template
+
+        or a string of the form: ``module://my.module.name``.
```
Comment (anntzer) [inline lib/matplotlib/__init__.py]: *this* value is not case-insensitive

**Comment by timhoffm** [comment]
Ha! Broke my own test :roll_eyes:

File: lib/matplotlib/__init__.py [hunk 2]
```diff
@@ -1311,16 +1311,23 @@ def use(arg, warn=True, force=False):
...
+
+        Note: Standard backend names are is case-insensitive here.
```
Comment (QuLogic) [inline lib/matplotlib/__init__.py]: s/are is/are/

**Pull request merged**